lms/static/sass/shared/_header.scss

```diff
@@ -145,6 +145,30 @@ header.global {
           }
         }
       }
+
+      .wrapper-more-actions {
+
+        // Some overrides and customization for the button
+        // in this particular use case
+        .button-more {
+          @include margin-left(-1px); // Don't want a double border on the side with the profile name
+          @include border-top-left-radius(0);
+          @include border-bottom-left-radius(0);
+          border-color: $gray-l3;
+          height: 34px;
+          line-height: 34px;
+          width: 34px;
+
+          .icon {
+            position: relative;
+            top: -3px;
+          }
+        }
+
+        .dropdown-menu {
```

Comment:
Why isn't this font-size set on the general rule instead of specific for the header?